universum/modules/artifact_collector.py

```diff
@@ -220,16 +211,13 @@ def move_artifact(self, path, is_report=False):
                     artifact_path = self.automation_server.artifact_path(self.artifact_dir, artifact_name)
                     self.collected_report_artifacts.add(artifact_path)
 
-    @make_block("Collecting artifacts", pass_errors=False)
-    def collect_artifacts(self):
-        self.reporter.add_block_to_report(self.structure.get_current_block())
-        for path in self.report_artifact_list:
-            name = "Collecting '" + os.path.basename(path) + "' for report"
-            self.structure.run_in_block(self.move_artifact, name, False, path, is_report=True)
-        self.reporter.report_artifacts(list(self.collected_report_artifacts))
-        for path in self.artifact_list:
-            name = "Collecting '" + os.path.basename(path) + "'"
-            self.structure.run_in_block(self.move_artifact, name, False, path)
+    def collect_step_artifacts(self, step):
```

Comment:
The function only accesses step.artifacts and step.report_artifacts.
I think you don't need to pass the step. Instead, you can directly pass artifacts to be collected.